### PR TITLE
GATE-2293 added teams logging settings to api

### PR DESCRIPTION
**Pull request by rexscaria**
A new api is added to teams accounts to change logging preferences for the proxy. This PR adds cloudflare-go helper methods for accessing these APIs.

## Has your change been tested?

unit tests with mock API response.

## Types of changes

What sort of change does your code introduce/modify?

- [ ] New feature for GET/PUT on teams logging preferences (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by jacobbednarz** [comment]
could you please pull in the latest master branch? there are some CI violations here that are showing in master but not on your fork.

**Comment by rexscaria** [comment]
The build is fixed. Thanks

File: teams_accounts.go [hunk 1]
```diff
@@ -112,6 +140,26 @@ func (api *API) TeamsAccountConfiguration(ctx context.Context, accountID string)
 	return teamsConfigResponse.Result, nil
 }
 
+// TeamsAccountLoggingConfiguration returns teams account logging configuration.
+//
+// API reference: TBA.
```
Comment (jacobbednarz) [inline teams_accounts.go:145]: i see GATE-2311 is targeting getting this publicly documented. should we wait until that lands for the public stability?

Comment (jacobbednarz) [reply teams_accounts.go:145]: confirmed the team is fine with this endpoint being used and will update the documentation following these changes.

**Pull request merged**